Source/Creation/Frame/Content/ID3FrameCreatorsChainFactory.swift

```diff
@@ -51,8 +51,15 @@ class ID3FrameCreatorsChainFactory {
                 paddingAdder: paddingAdder,
                 frameConfiguration: frameConfiguration)
         )
-
-// MARK: Specific Frame Creators (Text)
+        let frameFromCreditsListISO88591ContentCreator = ID3CreditsListFrameCreator(
+            frameContentSizeCalculator: frameContentSizeCalculator,
+            frameFlagsCreator: frameFlagsCreator,
+            stringToBytesAdapter: ID3ISO88591StringToByteAdapter(
```

Comment:
Am I using the correct StringToBytes adapter here? Because the test for this frame creator is coming out strangely. (see below.)

Reply:
Well, `ISO88591` matches between the names `frameFromCreditsListISO88591ContentCreator` and `ID3ISO88591StringToByteAdapter`. So if something is wrong, the problem is elsewhere.